portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path

```diff
@@ -58,7 +58,7 @@
 </tr>
 <tr>
 <td>DROPDOWN_MENU_ITEM</td>
-	<td>//div[contains(@class,'dropdown-menu') and contains(@class,'show')]//*[contains(@class,'dropdown-item') and contains(.,'${key_locale}')] | //div[contains(@class,'lfr-icon-menu-open') and contains(@class,'open')]//*[contains(@class,'dropdown-item') and contains(.,'${key_locale}')]</td>
+	<td>//div[contains(@class,'dropdown-menu') and contains(@class,'show')]//*[contains(@class,'dropdown-item') and contains(.,'${key_locale}')] | //div[contains(@class,'overlay-focused')]//div[contains(@class,'lfr-icon-menu-open')]//*[contains(@class,'dropdown-item') and contains(.,'${key_locale}')]</td>
```

Comment:
This path is using in several cases, have you confirmed that your changes will not affect them?

And this file has conflicts, please resolve first.